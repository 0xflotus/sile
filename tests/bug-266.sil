\begin[papersize=a6]{document}

Word\glue[width=1ex]to the wise
\font[size=20pt]

Word\glue[width=1ex]to the wise
\font[size=0.1pt]
\end{document}