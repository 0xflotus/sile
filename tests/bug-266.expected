<tests/bug-266.sil>
Set paper size 	297.6377985	419.5275636
Begin page
Mx 	34.8819
My 	28.5447
Set font 	Gentium;10;400;;normal;;LTR
T	55 79 82 68	(Word)
Mx 	62.1133
T	84 79	(to)
Mx 	73.2469
T	84 72 69	(the)
Mx 	89.4878
T	87 73 83 69	(wise)
Mx 	34.8819
My 	44.8279
Set font 	Gentium;20;400;;normal;;LTR
T	55 79 82 68	(Word)
Mx 	89.3448
T	84 79	(to)
Mx 	111.5958
T	84 72 69	(the)
Mx 	144.0617
T	87 73 83 69	(wise)
[1] Mx 	146.4727
My 	392.0445
Set font 	Gentium;10;400;;normal;;LTR
T	17	(1)
End page
Finish

